modules/node_modules/@frogpond/ccc-wpjson-feed/index.js

```diff
@@ -53,3 +53,11 @@ export function convertWpJsonItemToStory(item) {
 		title: JSDOM.fragment(item.title.rendered).textContent.trim(),
 	}
 }
+
+export function deprecatedWpJson() {
+	return [{
+		content: "",
+		excerpt: "Please update your app.",
+		title: "Deprecated endpoint",
```

Comment:
"This news source is no longer being updated." maybe?

Reply:
That is an improvement. If we provide a link to learn more, do our chevrons indicate they can learn more by tapping the row, or should we explicitly provide that in the excerpt?

> Please update your app
> This news source is no longer being updated. Tap to learn more about why you're seeing this message.

Reply:
I don't think we need to say "tap to learn more", imo.